main.go

```diff
@@ -86,6 +91,17 @@ func getConfigs(options *Options, userConfigDir string) []string {
 }
 
 func main() {
+	userConfigDir := filepath.Join(ExpandPath("~/"), ".config/smug")
+
+	if err := initConfigDir(userConfigDir); err != nil {
+		fmt.Fprintf(
+			os.Stderr,
+			"Cannot initialize config dir at ~/.config/smug : %q",
+			err.Error(),
+		)
+		os.Exit(1)
+	}
```

Comment:
I would simplify this part to just: 

```
if err := os.MkdirAll(path, 0o755); err != nil {
		fmt.Fprintf(
			os.Stderr,
			"Cannot initialize config dir at ~/.config/smug : %q",
			err.Error(),
		)
		os.Exit(1)
}
```

It should be no-op if directory already exists.